pipeline/profile-controller/base/configmap.yaml

```diff
@@ -22,26 +28,82 @@ data:
-                    "namespace": namespace
+                        "name": "mlpipeline-minio-artifact",
+                        "namespace": namespace,
                     },
                     "data": {
                         "accesskey": "bWluaW8=", # base64 for minio
-                        "secretkey": "bWluaW8xMjM=" # base64 for minio123
-                    }
+                        "secretkey": "bWluaW8xMjM=", # base64 for minio123
+                    },
                 },
                 {
                     "apiVersion": "v1",
                     "kind": "ConfigMap",
                     "metadata": {
-                    "name": "metadata-grpc-configmap",
-                    "namespace": namespace
+                        "name": "metadata-grpc-configmap",
+                        "namespace": namespace,
                     },
                     "data": {
-                    "METADATA_GRPC_SERVICE_HOST": "metadata-grpc-service.kubeflow",
-                    "METADATA_GRPC_SERVICE_PORT": "8080"
-                    }
-                }
+                        "METADATA_GRPC_SERVICE_HOST": "metadata-grpc-service.kubeflow",
+                        "METADATA_GRPC_SERVICE_PORT": "8080",
+                    },
+                },
+                {
+                    "apiVersion": "v1",
+                    "kind": "ServiceAccount",
+                    "metadata": {
+                        "name": "ml-pipeline-visualizationserver",
+                        "namespace": namespace,
+                    },
+                },
+                {
+                    "apiVersion": "apps/v1",
+                    "kind": "Deployment",
+                    "metadata": {
+                        "labels": {"app": "ml-pipeline-visualizationserver"},
+                        "name": "ml-pipeline-visualizationserver",
+                        "namespace": namespace,
+                    },
+                    "spec": {
+                        "selector": {
+                            "matchLabels": {"app": "ml-pipeline-visualizationserver"},
+                        },
+                        "template": {
+                            "metadata": {
+                                "labels": {"app": "ml-pipeline-visualizationserver"},
+                                "annotations": {"sidecar.istio.io/inject": "false"},
+                            },
+                            "spec": {
+                                "containers": [{
+                                    "image": "gcr.io/ml-pipeline/visualization-server:0.4.0",
+                                    "imagePullPolicy": "IfNotPresent",
+                                    "name": "ml-pipeline-visualizationserver",
+                                    "ports": [{"containerPort": 8888}],
+                                }],
+                                "serviceAccountName": "ml-pipeline-visualizationserver",
```

Comment:
I think we should use `default-editor` KSA right now, because visualization server needs to fetch data from GCS when visualizing TFX artifacts.

In the long run, as mentioned in the design, we should use `default-viewer` that has just read permission.

Reply:
I followed the the service account name used by standalone deployment, and I got the impression that GCS permission needs to be explicitly granted to an GSA, and user would need to manually create KSA-GSA binding for this service account to work.

Are you saying that `default-editor` already has the binding that allows GCS access? or we expect user to manually bind `default-editor` or `default-viewer` instead of using a third one just for visualization purpose?

Reply:
`default-editor` already has the binding. It is bound by profile controller using the GSA specified in `Profile` CR. I think that's the most seamless solution we can provide.

Reply:
SG, in that case I'll simply remove this dedicated SA. One last question, why not just `default-viewer` now? Is that because we don't have the proper binding for it yet?

Reply:
Exactly, `default-viewer` doesn't have proper binding.